airbyte-integrations/connectors/source-woocommerce/source_woocommerce/source.py

```diff
@@ -75,6 +100,7 @@ def limit(self):
     state_checkpoint_interval = limit
     # Setting the default cursor field for all streams
     cursor_field = "date_modified"
+    range_days = 15  # date range is set to 15 days, because for conversion_window_days default value is 14. Range less than 15 days will break the integration tests.
```

Comment:
where is this used?

Reply:
no, we are not using it anywhere. Removed it